cmd/baton-okta/config.go

```diff
@@ -19,6 +19,7 @@ var (
 	cacheTTI              = field.IntField("cache-tti", field.WithDescription("Response cache cleanup interval in seconds"), field.WithDefaultValue(60))
 	cacheTTL              = field.IntField("cache-ttl", field.WithDescription("Response cache time to live in seconds"), field.WithDefaultValue(300))
 	syncCustomRoles       = field.BoolField("sync-custom-roles", field.WithDescription("Enable syncing custom roles"), field.WithDefaultValue(false))
+	syncSecondaryEmails   = field.BoolField("sync-secondary-emails", field.WithDescription("Enable syncing secondary emails"), field.WithDefaultValue(true))
```

Comment:
You might want to invert this if you want to maintain syncing these emails as the default behavior. As it is, if you want to disable the syncing you'd have to pass `--sync-secondary-emails=false` vs `--disable-syncing-secondary-emails`.

Reply:
`--skip-secondary-emails`

Reply:
done!